net-mail/poppassd_ceti/poppassd_ceti-1.8.7.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pam
+
+# Long ago it was just "poppassd", but upstream now seems to have
+# settled on "poppassd-ceti" (instead of "poppassd_ceti" or no suffix).
+MY_PN="poppassd-ceti"
+MY_P="${MY_PN}-${PV}"
+S=${WORKDIR}/${MY_P}
+
+DESCRIPTION="Password change daemon with PAM support"
+HOMEPAGE="https://github.com/kravietz/poppassd-ceti"
+SRC_URI="https://github.com/kravietz/${MY_PN}/releases/download/v${PV}/${MY_P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="sys-libs/pam"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/doc-sudoAlternative-1.8.7.patch"
+)
+
+src_install() {
+	dodoc README.md
+
+	pamd_mimic_system poppassd auth account password
+
+	insinto /etc/xinetd.d
+	newins "${FILESDIR}"/poppassd.xinetd poppassd
+
+	insinto /usr/sbin
+	insopts -o root -g bin -m 500
```

Comment:
Mode 500 sounds like an executable, so any reason not to use `exeinto`, `exeopts` and `doexe`?